### PR TITLE
rename variable names

### PR DESCRIPTION
**Pull request by yungezz**
## Purpose

* ...
rename variable names

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## How to Test
*  Install ansible

```bash
$ pip install ansible[azure]
```

*  [Optional] Install ansible role

```bash
$ ansible-galaxy install Azure.azure_preview_modules
$ pip install -r ~/.ansible/roles/Azure.azure_preview_modules/files/requirements-azure.txt
```

*  Get the sample playbook

```
git clone https://github.com/Azure-Samples/ansible-playbooks.git
cd ansible-playbooks
git checkout [branch-name]
```

* Test the code

```bash
ansible-playbook <filename>.yml
```

## What to Check
Verify that the playbook is successfully run.

### CODE REVIEW

File: vm_create_image.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@
     vm_name: testvm
     vm_image_name: testvmimage
     location: eastus
-    admin_username: zim
+    admin_username: azureuser
```
Comment (zikalino) [inline vm_create_image.yml]: no! no! no! ;-)

Comment (yungezz) [reply vm_create_image.yml]: you can change your display name to azureuser :)

**Comment by Fred-sun** [comment]
@yungezz

**Comment by Fred-sun** [comment]
@yungezz Could you help to confirm whether this PR is merged or needs further review? Thanks!

**Comment by Fred-sun** [comment]
@yungezz

**Comment by Fred-sun** [comment]
@yungezz

**Pull request merged**